charts/intel-gpu-resource-driver/README.md

```diff
@@ -1,4 +1,5 @@
 # Dynamic Resource Allocation (DRA) Intel GPU Driver Helm Chart
+![Version: 0.4.0](https://img.shields.io/badge/Version-0.4.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: v0.4.0](https://img.shields.io/badge/AppVersion-v0.4.0-informational?style=flat-square)
```

Comment:
While badges seem popular, do we want image links to external 3rd party sites?

Only <10% of the current `README.md` files in this repo have them, and none in the top level chart directories:
```
$ git grep -l shields
charts/evi-hbase-0.8.3/charts/hbase/README.md
charts/evi-hbase-2.1.0/charts/hbase/README.md
charts/evi-minio-0.8.0/charts/minio/README.md
charts/evi-minio-0.8.1/charts/minio/README.md
charts/evi-minio-0.8.4/charts/minio/README.md
charts/evi-minio-2.1.0/charts/minio/README.md
charts/new-hbase-0.8.0/charts/hbase/README.md
```

Reply:
I am okay with both options, either have them or not

Reply:
removed